pkg/controller/node/nodecontroller.go

```diff
@@ -63,6 +64,18 @@ var (
 	// The minimum kubelet version for which the nodecontroller
 	// can safely flip pod.Status to NotReady.
 	podStatusReconciliationVersion = utilversion.MustParseSemantic("v1.2.0")
+
+	UnresponsiveTaintTemplate = &v1.Taint{
+		Key:    metav1.TaintNodeUnreachable,
+		Value:  "NodeControllerTaint",
```

Comment:
To allow user specify other 'unreachable' or 'notReady' taints. I'm not attached to this idea at all, so I can just remove this.